tools/secure_notebook.py

```diff
@@ -66,16 +78,33 @@ def create_self_signed_cert(cert_dir, keyfile, certfiile):
     except ConfigFileNotFound:
         config = Config()
 
-    config.NotebookApp.password = pw
+    yield config
 
     with io.open(os.path.join(jupyter_config_dir(), 'jupyter_notebook_config.json'), 'w') as f:
```

Comment:
If we're storing passwords, should we be trying to ensure that the config file is only user readable (i.e. chmod 600 on it)?